.github/pull_request_template.md

```diff
@@ -0,0 +1,18 @@
+## Problem
+
+_Describe the problem this PR solves_
+
+## Solution
+
+_Describe the solution this PR implements_
+
+## How To Test
+
+_Describe the steps required to test the changes_
+
+## Additional Checks
+
+- [ ] The PR includes one or several **stories** (if not possible, describe why)
+- [ ] The **documentation** is up to date
+
+Also, please make sure to read the [contributing guidelines](https://github.com/marmelab/react-admin#contributing).
```

Comment:
Good thing you added a PR template! \o/

But sections about the stories and contributing guidelines might not be applicable to this repo, so I'd remove them. Wdyt?

```suggestion
- [ ] The **documentation** is up to date
```